docgen/src/guide/React_native.md

```diff
@@ -6,9 +6,643 @@ category: guide
 navWeight: 35
 ---
 
-React InstantSearch is compatible with [react-native](https://facebook.github.io/react-native/).
+React InstantSearch is compatible with [react-native](https://facebook.github.io/react-native/)
+
+In this tutorial, you’ll learn how to:
+
+* install react-instantsearch in your [React Native](https://facebook.github.io/react-native/)  project
+* bootstrap an app using the <InstantSearch> component
+* display infinite results from Algolia
+* add widgets to filter the results
+* connect your own component to the search
+
+## Before we start
+React InstantSearch is meant to be used with Algolia.
+
+Therefore, you’ll need the credentials to an Algolia index. To ease this getting started, here are the credentials to an already configured index:
+
+* appId: latency
+* searchKey: 3d9875e51fbd20c7754e65422f7ce5e1
+* indexName: ikea
```

Comment:
On the documentation of Vue InstantSearch, no demo credentials were given IIRC, while they were in the docs first. 

Same for me though :)